### DISCUSSION TITLE
Indicator CHOCH

### DISCUSSION THREAD

**Discussion by ArtemHachaturov** [DaveSkender/Stock.Indicators#1201] [Q&A] [answered]
please tell me if there is an indicator in your list that is most suitable for the CHOCH inlicator and whether you plan to expand the list of inlicators, for example add Choch

**Comment by DaveSkender** [comment, marked as answer]
### re: new indicators, in general

We are collecting indicators in [the backlog](https://github.com/users/DaveSkender/projects/1) that meet:

- #1024.

>[!WARNING]
> The backlog is long and as of June 2024 I've been prioritizing work on v3 feature items and less on developing new indicators.  To be realistic, most new indicator requests either won't pass muster or may not get any of my attention for 6 months to a year, given all of my competing priorities.  With that said, create a PR with code!  Contributing code is a good way to get things prioritized.  Also, consider creating your own [custom indicators](https://dotnet.stockindicators.dev/custom-indicators/#content).

### re: CHOCH

I've not seen CHOCH before; looks interesting, like a combination of:

- #226, and
- #1202

If you have good links to the original source author's publications, with mathematical formulas, please share; but also please don't send links to code, TradingView, MT4, or YouTube videos!

**Comment by ArtemHachaturov** [comment]
CHoCH, BOS, and FLIP are important concepts in technical analysis that help traders identify and confirm trend changes, as well as determine key support and resistance levels. These indicators are used to make more informed decisions when entering and exiting positions, contributing to more effective risk management and increased profitability.

CHoCH (Change of Character) and BOS (Break of Structure) are widely used in the Smart Money strategy, which is based on analyzing the actions of large market participants.

Here's what I found: https://github.com/joshyattridge/smart-money-concepts
